test/extended/builds/revision.go

```diff
@@ -25,8 +25,15 @@ var _ = g.Describe("[builds] build have source revision metadata", func() {
 
 	g.Describe("started build", func() {
 		g.It("should contain source revision information", func() {
-			g.By("starting the build with --wait flag")
-			err := oc.Run("start-build").Args("sample-build", "--wait").Execute()
+			g.By("starting the build with")
+			err := oc.Run("start-build").Args("sample-build").Execute()
```

Comment:
Just got to the bottom of the PR and read your comment. Your preference has been noted and I will introduce the change.